compiler/passes/convert-uast.cpp

```diff
@@ -376,9 +581,27 @@ struct Converter {
     return ret;
   }
 
-  Expr* visit(const uast::OpCall* node) {
+  CallExpr* convertDmappedOp(const uast::OpCall* node) {
+    INT_ASSERT(node->numActuals() == 2);
+
+    CallExpr* ret = new CallExpr("chpl__distributed");
+
+    // Call takes actuals in reverse order.
+    for (int i = node->numActuals()-1; i >= 0; i--) {
+      Expr* actual = convertAST(node->actual(i));
+      INT_ASSERT(actual);
+      ret->insertAtTail(actual);
```

Comment:
You could also loop in the normal order and call insertAtHead, if that appeals more

Reply:
I thought about that, but wasn't sure if `insertAtHead` displaces the calledExpression. I guess I could always insert it at the end?

Reply:
it should leave the called expression alone